docs/1-Using-Fleet/11-Supported-browsers.md

```diff
@@ -0,0 +1,19 @@
+
+# Supported browsers
+
+Fleet supports the latest, stable releases of all major browsers and platforms. Specific minimum supported browser information is provided below.
+
+**Note:** This information also applies to [fleetdm.com](https://www.fleetdm.com).
+
+### Desktop
+
+- Chrome 29 on latest Windows OS where it is available
+- Firefox 28 on latest Windows OS where it is available
+- IE 11 on latest Windows OS where it is available
+- Edge 16 on latest Windows OS where it is available
+- Safari 13.x on latest macOS where it is available
```

Comment:
clarify that we only test with Windows - or better yet, remove the OS bit and instead clarify below that we test each of the above browsers on Windows whenever possible (because our engineering team primarily uses Macs)